spm_funcs.py

```diff
@@ -54,8 +54,9 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    img = nb.load(fname)
+    spm_vals = spm_global(img.get_fdata())
```

Comment:
Aha - not so fast - this returns the global across all 3D scans in the time-series - your job here was to return an array or list of global values, one for each 3D scan in the time-series.

Why not try fixing this on your computer, then pushing again to this same branch, to update the pull-request?